examples/graphics/area-lights.html

```diff
@@ -186,6 +183,9 @@
 
             var miniStats = new pcx.MiniStats(app);
 
+            // set the loaded area light LUT data
+            app.setAreaLightData(assetManifest[4].asset);
```

Comment:
I wonder if this should be called setAreaLightLuts perhaps? Data is too generic.

Reply:
sure - I can change this

Reply:
done